pkg/api/types.go

```diff
@@ -15,6 +15,8 @@ import (
 //  - raw steps that can be used to create custom and
 //    fine-grained build flows
 type ReleaseBuildConfiguration struct {
+	Metadata Metadata `json:"generated_metadata"`
```

Comment:
Would you mind changing the json tag to `zz_generated_metadata`, resembling how the kube generators name files? It has the advantage that it shows up at the end of the document after generation and thus does not split up the part that is relevant for humans